docs/applications/big-data/big-data-in-the-linode-cloud-streaming-data-processing-with-apache-storm.md

```diff
@@ -38,45 +38,48 @@ This guide explains how to create Storm clusters on the Linode cloud using a set
+-  The Cluster Manager Linode can have either Ubuntu 14.04 LTS or Debian 8 installed.
+-  A Zookeeper or Storm cluster can have either Ubuntu 14.04 LTS or Debian 8 installed on its nodes. Its distribution does not need to be the same one as the one installed on the Cluster Manager Linode.
 
-### Get a Linode API Key
-
-Follow the steps in [Generating an API Key](/docs/platform/api/api-key) and save your key securely. It will be entered into configuration files in upcoming steps.
-
-If the key expires or is removed, remember to create a new one and update the *api_env_linode.conf* API environment configuration file on Cluster Manager Linode. This will be explained further in the next section.
+{: .note}
+>The steps in this guide and in the baSH scripts referenced require root privileges. Be sure to run the steps below as `root`. For more information on privileges, see our [Users and Groups](/docs/tools-reference/linux-users-and-groups) guide.
 
 ### Naming Conventions
 
 Throughout this guide, we will use the following names as examples that refer to the images and clusters we will be creating:
 
--   `zk-image1` - Zookeeper image
--   `zk-cluster1` - Zookeeper cluster
--   `storm-image1` - Storm image
--   `storm-cluster1` - Storm cluster
+-  `zk-image1` - Zookeeper image
+-  `zk-cluster1` - Zookeeper cluster
+-  `storm-image1` - Storm image
+-  `storm-cluster1` - Storm cluster
 
 These are the names we'll use, but you are welcome to choose your own when creating your own images and clusters. This guide will use these names in all example commands, so be sure to substitute your own names where applicable.
 
+### Get a Linode API Key
+
+Follow the steps in [Generating an API Key](/docs/platform/api/api-key) and save your key securely. It will be entered into configuration files in upcoming steps.
+
+If the key expires or is removed, remember to create a new one and update the `api_env_linode.conf` API environment configuration file on Cluster Manager Linode. This will be explained further in the next section.
+
 ## Set Up the Cluster Manager Linode
 
 The first step is setting up a central *Cluster Manager* Linode to store details of all Storm clusters, and enable authorized users to create, manage or access those clusters. 
 
 You can set up multiple Cluster Manager nodes if necessary to satisfy organizational or security needs. For example, two departments can each have their own Cluster Manager nodes to manage their respective clusters and keep data access restricted to their own members.
 
-1.  The scripts communicate with Linode's API using Python. Install Git, Python 2.7 and curl on your Ubuntu 14.04 LTS or Debian 8 workstation:
+1.  The scripts communicate with Linode's API using Python. Install Git, Python 2.7 and curl on your Ubuntu 14.04 LTS or Debian 8 Linode:
 
-        sudo apt-get install python2.7 curl git
+        apt-get install python2.7 curl git
```

Comment:
I feel keeping the sudo is better, because most Ubuntu / Debian users don't log in as 'root' on their workstations, but log in as a normal user with sudo privilege. They'll have to type 'sudo' anyway for apt-get install.